### PR TITLE
Add : Utility files

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

**Review by Hamdrive — COMMENTED** [review]
Nice implementations overall, good use of logic for handling cart/wishlist
Loader library used is good, gives a better UI

File: src/frontend/context/cartContext.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        }).catch((error) => {
+            console.log(error.response.data)
+        })
+    }
+
+
+    const addItem = (product) => {
+
+        // if (token != null) {
+        // setisLogged(true)
```
Comment (krate27) [inline src/frontend/context/cartContext.js:35]: Remove Unwanted commented code.

**Review by krate27 — APPROVED** [review]
Things look good in code. Could use a good formatter in jsx code.
Overall its good to go from my side.

File: src/frontend/context/cartContext.js (hunk 1 above)
Comment (ZuberDunge) [reply src/frontend/context/cartContext.js:35]: done!

**Pull request merged**